src/core/src/sys/interactions.rs

```diff
@@ -278,6 +299,34 @@ mod test {
     }
 
     #[test]
+    fn sorting_pairs() {
+        assert_eq!(sort_pair(Kind(0), Kind(1)), sort_pair(Kind(1), Kind(0)));
+        assert_eq!(sort_pair(Kind(125), Kind(0)), sort_pair(Kind(0), Kind(125)));
+    }
+
+    #[test]
+    fn sorting_angles() {
+        assert_eq!(sort_angle(Kind(1), Kind(0), Kind(2)), sort_angle(Kind(2), Kind(0), Kind(1)));
```

Comment:
Yes, naming is bad. This function is more about normalisation than sorting. We don't actually care that they are sorted, only that equivalent bonds/angles.dihedrals have an unique representation.